Products/ZenUI3/browser/resources/js/zenoss/Reports.js

```diff
@@ -158,12 +158,19 @@ Ext.define('Zenoss.ReportTreePanel', {
             msg: message,
             fn: function(buttonid) {
                 if (buttonid==='ok') {
+                    report_tree._deleteChilds();
                     report_tree._deleteSelectedNode();
                 }
             },
             buttons: Ext.MessageBox.OKCANCEL
         });
     },
+    _deleteChilds: function () {
+        var node = this.getSelectionModel().getSelectedNode();
+        while(node.firstChild) {
+            node.removeChild(node.firstChild);
+        }
```

Comment:
Suborganizers will be deleted as well. This function is only for removing JS's objects.